### ISSUE TITLE
小文件过多，文件数超上限，导致vol分配大量dp，节点压力过大

### ISSUE THREAD

**Issue by vnline** [chubaofs/chubaofs#1103]
**Describe the bug**
vol 文件数量规模大，vol大量dp超过20000文件数上限，dp变成只读，分割新的dp。过多的vol引起节点压力过太，性能下降严重。

Data partition count : 1856
Read only Data partition count: 1830

**Environment**
ChubaoFS Server
Version : v2.2.1
Branch  : master
Commit  : 00ebf481a2f78d94607c72cb69d437dd74cf23c4
Build   : go1.14.3 linux amd64 2020-09-08 19:05

**Comment by xrefft** [comment]
```shell
du  * |awk 'BEGIN{count=0;size=0;} \
     {count = count + 1; size = size + $1/1024;} \
     END{print "Total count " count; \
       print "Total size " size/1024 " GB" ; \
       print "Avg size " size / count "MB"}'
```
请使用上述脚本计算某个过载dp里extent的平均大小，这样我们可以更好的帮您定位问题。

(You can use the above shell to calculate the average extent file size in one of those overloaded datapartitions. So that we can help you to locate the problem faster.)

**Comment by Victor1319** [comment]
hello, please refer PR `https://github.com/chubaofs/chubaofs/pull/1106`, it may be useful to resolve your  problem.

**Comment by shuoranliu** [comment]
fixed by #1106
